### PR TITLE
ref(app-store-connect): Add app store connect modal url & expiration alert

### PR DESCRIPTION
**Pull request by priscilawebdev**
- Add "App Store Connect session expire" alerts on the issue details page and project details page 
   Preview:

image

 - Open/Close Debug Files - Custom Repositories according to the URL parameters
 - Fix annoying bug that occurs when the request returns 401 - The  page is being reloaded

this PR depends on https://github.com/getsentry/sentry/pull/26154  to be merged

### CODE REVIEW

**Review by matejminar — APPROVED** [review]

File: src/sentry/utils/prompts.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "releases": {"required_fields": ["organization_id", "project_id"]},
     "suspect_commits": {"required_fields": ["organization_id", "project_id"]},
     "alert_stream": {"required_fields": ["organization_id"]},
+    "app_store_connect_updates": {"required_fields": ["organization_id", "project_id"]},
```
Comment (flub) [inline src/sentry/utils/prompts.py:5]: I have no idea what this is.  But assume it's good...

File: src/sentry/api/exceptions.py [hunk 2]
```diff
@@ -97,6 +97,24 @@ class TwoFactorRequired(SentryAPIException):
     message = "Organization requires two-factor authentication to be enabled"
 
 
+class AppConnectAuthenticationError(SentryAPIException):
+    status_code = status.HTTP_401_UNAUTHORIZED
+    code = "app-connect-authentication-error"
+    message = "App connect authentication error"
```
Comment (flub) [inline src/sentry/api/exceptions.py:103]: oh, this is a much nicer way of returning errors.  nice!

File: src/sentry/utils/prompts.py (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/utils/prompts.py:5]: this is to suspend an alert message for a certain time

**Review by flub — APPROVED** [review]
python code lgtm.  did not review any typescript 😄

**Pull request merged**